src/breeze_carbon/actions/emails/render.cr

```diff
@@ -23,6 +25,23 @@ class BreezeCarbon::Emails::Render < Breeze::BreezeAction
     end
   end
 
+  # Scan all links in the email, and add the target _blank
+  # This is so you can click on the links, and they don't
+  # open up in the iframe.
+  private def parse_body_and_replace_links(body : String)
+    body.scan(/<a\s*([^>]+)>/) do |match|
+      if data = match[1]?
+        if data.includes?(%(target=))
+          # target is already included
+        else
+          body = body.sub("<a #{data}", %(<a #{data} target="_blank"))
```

Comment:
I do a `sub` instead of `gsub` here because `gsub` causes it to add 2 targets if there's more than 1 of the same link (i.e. home link).